app/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,11 @@
     <!-- Strings used for fragments for navigation -->
     <string name="next">Next</string>
     <string name="previous">Previous</string>
+
+    <string name="add_transaction">Add a transaction</string>
```

Comment:
can you read up on how strings.xml is generally maintained? it might in an alphabetical order, or depending on usage within activities. We can have sections preceded by comments, like the default ones